### PR TITLE
fix and tag VPN links

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

**Comment by groovecoder** [comment]
@maxxcrawford I can't seem to fix this lack of spacing between the "like" and "Mozilla VPN" link ?

image

File: template-helpers/articles.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ const articleCopy = {
               "Use unique, strong passwords for every account",
               "Use a password manager to remember all your passwords for you",
               "Turn on two-factor authentication for an extra layer of security",
-              "Use a VPN (Virtual Private Network) when using public Wi-Fi",
+              "Use a VPN (like <a target='_blank' href='https://vpn.mozilla.org/?utm_source=monitor.firefox.com&utm_medium=referral&utm_campaign=monitor-security-tips'> Mozilla VPN</a>) when using public Wi-Fi",
```
Comment (pdehaan) [inline template-helpers/articles.js:240]: nit: this has a space before " Mozilla VPN", which will cause a semi-awkward, stray underline when hovering over a link.

**Comment by maxxcrawford** [comment]
> @maxxcrawford I can't seem to fix this lack of spacing between the "like" and "Mozilla VPN" link ?
> 
> image

I'm not seeing this issue: 

Chrome (Stable/89): 
image


Firefox (Nightly/89.0a1): 
image

File: template-helpers/articles.js (hunk 1 above)
Comment (maxxcrawford) [inline template-helpers/articles.js:240]: Maybe not the best way, but putting a non-breaking space (`&nbsp;`) here totally works. 

```suggestion
              "Use a VPN (like&nbsp;<a target='_blank' href='https://vpn.mozilla.org/?utm_source=monitor.firefox.com&utm_medium=referral&utm_campaign=monitor-security-tips'> Mozilla VPN</a>) when using public Wi-Fi",
```

**Review by maxxcrawford — CHANGES_REQUESTED** [review]

**Comment by groovecoder** [comment]
Thanks. Fixed.

**Review by maxxcrawford — APPROVED** [review]

**Pull request merged**